src/utils/clearPeopleList.ts

```diff
@@ -0,0 +1,25 @@
+import { Person } from '../types';
+import { sortPeople } from './sortPeople';
+
+export function clearPeopleList(
+  people: Person[],
+  centuries: string[],
+  sex: string | null,
+  query: string | null,
+  sort: string | null,
+  order: string | null,
+) {
+  const visiblePeople = people.filter(
+    person =>
+      (!sex || person.sex === sex) &&
+      (person.name.includes(query || '') ||
+        person.motherName?.includes(query || '') ||
+        person.fatherName?.includes(query || '')) &&
+      (!centuries.length ||
+        centuries.includes(Math.ceil(person.born / 100).toString())),
+  );
+
+  const sortedPeople = sortPeople(sort, order, visiblePeople);
+
+  return sortedPeople;
```

Comment:
```suggestion
  return sort ? sortPeople(sort, order, visiblePeople)  :  visiblePeople;
```
Do we really need to call the sort function even if we don't have any sorting parameters?